### PR TITLE
Enhance documentation of translations.

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

File: documenting.rst [hunk 1]
```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+
+
+Starting a new translation
+--------------------------
+
+First subscribe to the `doc-sig
+<https://mail.python.org/mailman/listinfo/doc-sig>`_ mailing list,
+introduce yourself and the translation you're starting.
```
Comment (Seluj78) [inline documenting.rst]:
```suggestion
and introduce yourself and the translation you're starting.
```

File: documenting.rst [hunk 2]
```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+
+Then you can bootstrap your new translation by using our `cookiecutter
+<https://github.com/JulienPalard/python-docs-cookiecutter>`__.
```
Comment (Seluj78) [inline documenting.rst]: Should this be moved to `github.com/python/python-docs-cookiecutter`?

File: documenting.rst [hunk 3]
```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+PEP 545: TL;DR
+--------------
+
+The :PEP:`545` may be long to read, so here are the interesting points:
+
+- Each translation is assigned an appropriate lowercased language tag,
+  with an optional region subtag (if needed), glued with a dash, like
+  ``pt-br`` or ``fr``.
+
+- Translations files are hosted on
```
Comment (Seluj78) [inline documenting.rst]: Well tbh only the ones advanced enough, not all of them 🤔 how can we reflect that ?

File: documenting.rst [hunk 4]
```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+
+Which version of Python documentation should be translated?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Concensus is to work on current stable, you can then propagate your
+translation from a branch to another using `pomerge
+<https://pypi.org/p/pomerge>`__.
+
+
+Are there some tools to help in managing the repo?
```
Comment (Seluj78) [inline documenting.rst]: Should those tools be moved to be hosted on `github.com/python/xxx` ? Or is it useless ?

File: documenting.rst [hunk 5]
```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Italian (it) |                    | `mail <https://mail.python.org/pipermail/|
+|              |                    | doc-sig/2019-April/004114.html>`__       |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Japanese     | `Inada Naoki       | `github <https://github.com/python/      |
+| (ja)         | (methane) <https:/ | python-docs-ja/>`__                      |
+|              | /bugs.python.org/u |                                          |
+|              | ser10371>`__       | `doc <https://docs.python.org/ja/>`__    |
```
Comment (methane) [inline documenting.rst]: Please change contact to @cocoatomo https://bugs.python.org/user19001

Comment (JulienPalard) [reply documenting.rst]: OK done.

File: documenting.rst (hunk 1 above)
Comment (JulienPalard) [reply documenting.rst]: Is it mandatory? I kind of prefer it without the and.

File: documenting.rst (hunk 2 above)
Comment (JulienPalard) [reply documenting.rst]: Not sure it's necessary, let's not change everything today.

File: documenting.rst (hunk 4 above)
Comment (JulienPalard) [reply documenting.rst]: Same, there's OK where they are.

File: documenting.rst (hunk 1 above)
Comment (terryjreedy) [reply documenting.rst]: Yes.  Or divide into two sentences. 'mailing list.  Next introduce'

File: documenting.rst [hunk 6]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+
+Then you can bootstrap your new translation by using our `cookiecutter
+<https://github.com/JulienPalard/python-docs-cookiecutter>`__.
+
+The important steps looks like this:
+
+- Create the github repo (anywhere), with the right hierarchy (using the
+  cookiecutter).
+- Gather people to help you translating, you can't do it alone.
+- You can use any tool to translate, as long as you can synchronize with git,
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
- You can use any tool to translate, as long as you can synchronize with git.
```

File: documenting.rst [hunk 7]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+  some are using Transifex, some are using only github, you can choose another
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
  Some are using Transifex, some are using only github, or you can choose another
```

File: documenting.rst [hunk 8]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+  way, it's up to you.
+- Ensure we updated this page to reflect your work and progress, either via a
+  PR, or by asking on the doc-sig mailing list.
+- When ``tutorial/``, ``library/stdtypes`` and ``library/functions``
+  are complete, ask on doc-sig for your language to be added in the
+  language picker on docs.python.org.
+
+
+PEP 545: TL;DR
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
PEP 545 summary:
```

File: documenting.rst [hunk 9]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+--------------
+
+The :PEP:`545` may be long to read, so here are the interesting points:
+
+- Each translation is assigned an appropriate lowercased language tag,
+  with an optional region subtag (if needed), glued with a dash, like
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
  with an optional region subtag, joined with a dash, like
```
'optional' and 'if needed' are redundant.

File: documenting.rst [hunk 10]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+  PR, or by asking on the doc-sig mailing list.
+- When ``tutorial/``, ``library/stdtypes`` and ``library/functions``
+  are complete, ask on doc-sig for your language to be added in the
+  language picker on docs.python.org.
+
+
+PEP 545: TL;DR
+--------------
+
+The :PEP:`545` may be long to read, so here are the interesting points:
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
Here are the essential points of :PEP:`545`:
```

File: documenting.rst [hunk 11]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+- `powrap <https://pypi.org/p/powrap>`__ to rewrap the ``.po`` files
+  before committing, this helps keeping git diffs short.
+- `potodo <https://pypi.org/p/potodo>`__ to list what needs to be translated.
+
+
+
+How a coordinator is elected?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no election, each country have to short this out, here's some
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
There is no election, each country have to sort this out.  Here are some suggestions.
```
spelling

File: documenting.rst [hunk 12]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+rule if it helps:
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
```
ie, delete

File: documenting.rst (hunk 1 above)
Comment (Seluj78) [reply documenting.rst]: It just feels better with it IMO but whatever :)

File: documenting.rst (hunk 2 above)
Comment (Seluj78) [reply documenting.rst]: Just asking but fine with me

File: documenting.rst [hunk 13]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+
+-  Coordinator requests are to be public on doc-sig mailing list.
+-  If the given language have a native core dev, the core dev have its
+   word on the choice.
+-  If one want to become coordinator for its native language, and shows
+   motivation by translating and building a community, it will be named
+   coordinator.
```
Comment (terryjreedy) [inline documenting.rst]: Suggestion for sentence.
Anyone who wants ... and shows ... will be ... coordinatior.

File: documenting.rst [hunk 14]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+-  In case of concurrency between two persons, no one will sort this out
+   for you, it’s open to you to organize a local election or whatever
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
   for you.  It is up to you two to organize a local election or whatever
```

File: documenting.rst [hunk 15]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+Ask on doc-sig, or better, make a PR on the `devguide
+<https://github.com/python/devguide/>`__.
+
+
+I have a translation, but not on git, what should I do?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Just ask for help on the doc-sig mailing list and our python-fu, git-fu
+and bash-fu combined will help you create an appropriate repository. If
+you use a tool like transifex don’t worry keeping them in sync is not
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
you use a tool like transifex don’t worry.  Keeping them in sync is not
```

File: documenting.rst [hunk 16]
```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
...
+No, inside the ``github.com/python`` organization we’ll all have the
+exact same hierarchy so bots will be able to build all of our
+translations. So you may have to convert from one hierarchy to another.
+Ask for help on the doc-sig mailing list if you’re not sure on how to do
+it.
+
+What hierarchy should I use in my github repository?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As for every projects we have a *branch* per version, we store ``po``
```
Comment (terryjreedy) [inline documenting.rst]:
```suggestion
As for every project, we have a *branch* per version.  We store ``po``
```

File: documenting.rst (hunk 11 above)
Comment (Seluj78) [reply documenting.rst]:
```suggestion
There is no election, each country have to sort this out. Here are some suggestions
```
Maybe no need for two spaces after the `.` ? 😉

File: documenting.rst (hunk 1 above)
Comment (JulienPalard) [reply documenting.rst]: OK :) thanks

File: documenting.rst (hunk 13 above)
Comment (JulienPalard) [reply documenting.rst]: Ahh thanks, I disliked this one but was unable to find better :(

**Comment by JulienPalard** [comment]
@terryjreedy @methane @Seluj78 thanks for the review :)))))

**Pull request merged**